src/js/utils/MetronomeUtil.js

```diff
@@ -91,5 +91,36 @@ module.exports = {
     });
 
     return rootTree;
+  },
+
+  parseJob(job) {
+    let {history}= job;
+
+    if (history == null) {
+
```

Comment:
As I wrote in the other PR, there the according eslint-rule treats both cases as valid.
So I cannot fulfil your request.